trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/generate_VD_helpers.py

```diff
@@ -0,0 +1,172 @@
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift
+# generate_VD_helpers.py
+# 
+# Created:  Jun 2021, A. Blaufox
+# Modified: 
+#           
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# package imports 
+import numpy as np
+
+# ----------------------------------------------------------------------
+#  postprocess_VD
+# ----------------------------------------------------------------------
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift    
+def postprocess_VD(VD, settings):
+    """ 
+    Recomputes data about the VD. Should be called any time VD panel 
+    values (e.g. VD.XA1, VD.XCH, etc) are changed.
+
+    Assumptions: 
+
+    Source:  
+
+
+    Inputs: 
+    VD       - vehicle vortex distribution                    [Unitless] 
+    settings - Vortex_Lattice.settings                        [Unitless] 
+    
+    Outputs:      
+    VD       - vehicle vortex distribution                    [Unitless] 
+
+
+    Properties Used:
+    N/A
+    """     
+    #unpack
+    precision  = settings.floating_point_precision   
+    LE_ind     = VD.leading_edge_indices
+    TE_ind     = VD.trailing_edge_indices
+    strip_n_cw = VD.panels_per_strip[LE_ind]
+    
+    # @Emilio: assumes last VLM_wing is the last to get discretized & thus has highest surface_ID. 
+    #          This is correct for now, but do we want to continue to assume this, or should this be 
+    #           rewritten to allow any order of wings
+    last_wing_ID = list(VD.VLM_wings.values())[-1].surface_ID
```

Comment:
I think it's a fair assumption. If you add another wing you'd need to rediscretize.

Reply:
cool, I'll remove most of that comment and also bring into into the docstring assumptions

Reply:
done